src/main/java/com/denizenscript/ddiscordbot/objects/DiscordUserTag.java

```diff
@@ -413,6 +413,20 @@ public static void register() {
             return new ElementTag(activities.get(0).getUrl());
         });
 
+        // <--[tag]
+        // @attribute <DiscordUserTag.display_name>
+        // @returns ElementTag
+        // @plugin dDiscordBot
+        // @description
+        // Returns the global display name of the user.

```

Comment:
Meta needs to mention it might not return anything (which is usually just `, if any` at the end)